core/dao/project.go

```diff
@@ -336,9 +340,59 @@ func (c Config) GetProjectsByPath(dirs []string) ([]Project, error) {
 		// Variable use to check that all dirs are matched
 		var numMatched = 0
 		for _, dir := range dirs {
-			if strings.Contains(project.RelPath, dir) {
+
+			matchPath := func(dir string, path string) (bool, error) {
```

Comment:
You are passing in path but not using it. Should be project.RelPath?

Reply:
@alajmo My bad, indeed.

Here's a fix: https://github.com/alajmo/mani/pull/102/commits/dc7d0196d6179721c70a3f07c96b3274268a60d1